packages/babel-plugin-syntax-flow/src/index.js

```diff
@@ -1,11 +1,15 @@
 import { declare } from "@babel/helper-plugin-utils";
 
-export default declare(api => {
+export default declare((api, options) => {
   api.assertVersion(7);
 
+  // When enabled and plugins includes flow, all files should be parsed as if
+  // the @flow pragma was provided.
+  const { all } = options;
```

Comment:
@samwgoldman In our Babel team meeting today, we talked about how it would be nice to set this automatically based on the user's `.flowconfig` to avoid having to duplicate the value in two places. I have a few questions related to that:

1. Does flow allow multiple `.flowconfig` files, or can we just read the one in the project root? And along those lines, do you expect published npm libraries to then include a `.flowconfig` if they are using `all`, since otherwise it would publish without the flow pragma comments?
2. Is there a JS library we can use for parsing Flow's config?

Reply:
Alright, if there's no parser then we don't really have any options for now, but it does seem unfortunate that Babel needs to be told separately what type of file it is parsing.

Reply:
It's a pretty simple format. If you just want to check for the `all` option I imagine it wouldn't be hard to reimplement. I would think that the other problems are bigger roadblocks.